tendermint-rs/CHANGES.md

```diff
@@ -1,3 +1,13 @@
+## 0.1.6 (2019-01-23)
+
+This release is compatible with tendermint [v0.29]
+
+- Update to x25519-dalek v0.4.4 (#158)
+- Consistent ordering of `BlockID` and `Timestamps` in vote and proposal messages (#159)
+- Remove `PoisonPillMsg` previously used to shut-down the kms (#162)
```

Comment:
Would suggest bumping to `0.2` for a change like this, otherwise this will break builds for `tmkms` v0.2 which depend on [~v0.1](https://github.com/tendermint/kms/blob/master/Cargo.toml#L39).

Reply:
Thanks! Makes sense: https://github.com/tendermint/kms/pull/163/commits/a9cd131a20146fc7f8feba70b84c9acbedeb0834